doc/md_src_files/060_decision-trees.md

```diff
@@ -174,9 +187,53 @@ Thus, 60 unique combinations of decision values is the point at which a decision
 SSVC trees should be identifiable by name and version. A tree name is simply a short descriptive label for the tree derived from the stakeholder and/or function the tree is intended for. Tree versions are expected to share the major and minor version numbers with the SSVC version in which their decision points are defined. Revisions should increment the patch number. For example: “Applier Tree v1.1.0” would be the identity of the version of the Applier Tree as published in version 1.1 of SSVC.
 “Coordinator Publish Tree v2.0.3” would be the identity of a future revision of the Coordinator Publish Tree as described in this document. The terms “major”, “minor”, and “patch” with respect to version numbering are intended to be consistent with [Semantic Versioning 2.0.0](https://semver.org/spec/v2.0.0.html).
 
+### Sharing Trees With Others
+
+Communities of shared interest may desire to share information about decision points or even create custom trees to share within their community.
+Examples include:
+- an Information Sharing and Analysis Organization (ISAO) within a critical infrastructure sector might want to define a custom decision point relevant to their constituents' regulatory compliance.
+- a corporate Computer Security Incident Response Team (CSIRT) might choose to adjust decision priorities for an existing tree for use by its subsidiaries.
+- a government department might define a separate tree using existing decision points to address a particular governance process within their constituent agencies.
+- a regional coordinator might want to produce decision point information as a product of its threat analysis work and provide this information to its constituency in an advisory. 
+
+In these and other scenarios, there are two scopes to consider:
+1. Decision Point Scope
+2. Decision Tree Scope
+
+#### Decision Point Scope
+
+Each decision point defined in this document has a characteristic scope, either *global* or *local*.
+
+- *Globally scoped decision points* describe the state of the world outside the decision maker's environment. 
+They form the background context in which the stakeholder is making prioritization decisions. 
+Nearly all stakeholders should agree on the assignment of specific values to these decision points.
+- *Locally scoped decision points* are expected to be contextual to some decision makers.
+Mission Impact is one such example.
+Information about a locally scoped decision point can still be inherited by others using the same tree. 
+For example in the corporate CSIRT scenario above, the System Exposure value might be consistent across all subsidiaries for a centrally managed service.
+
```

Comment:
For example, regulatory or uptime impacts

Use terminology like locally -> stakeholder-specific, global -> "objective"?

Reply:
This comment has been addressed as of e223d7e803df2471afb2e700b7fe7a8b4d77d287